templates/house_view/mine_page.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<link rel="stylesheet" type="text/css" href="{% static 'css/mine_house_and_expenses.css' %}" />
+<body>
+  <div class="expenses_table-container">{% include 'house_view/expenses_table.html' %}</div>
+  <div class="house_table-container">{% include 'house_view/house_table.html' %}</div>
+  <div class="add-expense-container">{% include 'house_view/add_expense.html' %}</div>
+</body>
+{% endblock %}
```

Comment:
Missing empty line at the end of the file

Reply:
fixed